packages/wdio-utils/src/test-framework/testFnWrapper.ts

```diff
@@ -76,7 +76,7 @@ export const testFrameworkFnWrapper = async function (
     const testStart = Date.now()
     try {
         result = await promise
-    } catch (err) {
+    } catch (err: any) {
```

Comment:
```suggestion
    } catch (err: Error) {
```

Reply:
Catch clause variable type annotation must be 'any' or 'unknown' if specified.